source/content/drush-versions.md

```diff
@@ -54,21 +60,22 @@ Always use Drush 8 with Drupal 7 and Drupal 6 sites, as Drush 9 only works on Dr
 | Drush 7       | PHP 5.3.0+          |
 | Drush 8       | PHP 5.4.5+          |
 | Drush 9       | PHP 5.6.0+          |
+| Drush 10      | PHP 7.1.+           |
 
 See our guide on [Upgrading PHP Versions](/php-versions/).
 
 ## Troubleshooting
 
-Sometimes even after updating the drush version in `pantheon.yml`, the correct version of drush is not called. This is usually caused by an outdated configuration file, `policy.drush.inc`, in your local `~/.drush` directory, overriding `pantheon.yml`. Remove the file, or comment out its contents, to resolve.
+Sometimes even after updating the Drush version in `pantheon.yml`, the correct version of Drush is not called. This is usually caused by an outdated configuration file, `policy.drush.inc`, in your local `~/.drush` directory, overriding `pantheon.yml`. Remove the file, or comment out its contents to resolve.
 
 ### Site-local Drush Usage
-If you need to specify a minor version or a version not available on the platform, you can add a site-local installation of Drush to your repository. This will redispatch Pantheon's platform Drush to the site-local installation. Do not select any major version of Drush lower than 8.2.3 or 9.6.0. In general, you will usually be better off not including a site-local Drush in your Pantheon site, as using the Pantheon-provided Drush will ensure that you are using the recommended and most-tested version for the Pantheon platform. Occasionally, using a site-local Drush may be necessary for avoiding conflicts with your site's dependencies.
+If you need to specify a minor version or a version not available on the platform, you can add a site-local installation of Drush to your repository. This will re-dispatch Pantheon's platform Drush to the site-local installation. Do not select any major version of Drush lower than 8.2.3 or 9.6.0. In general, you will usually be better off not including a site-local Drush in your Pantheon site, as using the Pantheon-provided Drush will ensure that you are using the recommended and most-tested version for the Pantheon platform. Occasionally, using a site-local Drush may be necessary in order to avoid conflicts with your site's dependencies.
```

Comment:
This no longer re-dispatches; Pantheon will call the site-local Drush directly.

Do not select a version of Drush lower than 8.3.3 or 9.7.1.

Reply:
... or lower than 10.2.0.